docs/workflows/standalone/dorado_basecalling.md

```diff
@@ -0,0 +1,136 @@
+    Go to the **"Data"** tab in your Terra workspace. Click **"Upload Files"** and select your `POD5` files for upload. Confirm the upload process and wait for the files to be uploaded.
+
+    **Step 2: Copy the GCS Path**
+    After the upload is complete, Right-click the collection name and select "Copy link address."
+
+    ![Data uploader](../../assets/figures/dorado_pod5_data_upload.png)
+
+    **Step 3: Paste the GCS Path into the Workflow Input**
+    Open the workflow configuration screen in Terra. Paste the copied GCS path into the `pod5_bucket_path` input field for the Dorado Basecalling Workflow.
+    
+    ![Data uploader](../../assets/figures/dorado_input_seelction.png)
+
+!!! info "Detailed Input Information"
+    - **dorado_model**: If set to 'sup', 'hac', or 'fast', the workflow will run with automatic model selection. If a full model name is provided, Dorado will use that model directly.
+    - **fastq_file_name**: This will serve as a prefix for the output FASTQ files. For example, if you provide `project001-`, the resulting files will be named `project001-barcodeXX.fastq.gz`.
+    - **kit_name**: Ensure the correct kit name is provided, as it determines the barcoding and adapter trimming behavior.
+    - **fastq_upload_path**: This is the folder path in Terra where the final FASTQ files will be transferred for further analysis. Ensure the path matches your Terra workspace bucket GSURI. Please visit the "Dashboard" tab of your Terra workspace which provides a link to the bucket in the "Cloud Information" drop-down panel. The root GSURI for Terra workspace buckets generally start with `gs://fc-` followed by a unique identifier.
+
+!!! tip "File Naming Guidelines"
+    - **Avoid special characters**: Do not include special characters such as underscores (`_`), periods (`.`), or any non-alphanumeric symbols in the `fastq_file_name` prefix, as these can interfere with proper file parsing and sample name recognition.
+    - **Use a clear, simple prefix**: The prefix `projectname` will automatically append identifiers like `-barcodeXX.fastq.gz` or `-unclassified.fastq.gz` to name each output file, ensuring each one is distinct.
+
+    ### Examples:
+
+    - **Accepted Prefix**: `projectname-barcode01.fastq.gz`
+    - **Not Recommended**: `projectname_2024_test-barcode01.fastq.gz` (would recognize only `projectname` as the sample name, leading to ambiguity with multiple files).
+
+### Workflow Tasks
+
+This workflow is composed of several tasks to process, basecall, and analyze Oxford Nanopore `POD5` files:
+
+??? task "`Transfer POD5 Files`: Transfers `POD5` files to Terra"
+    Lists `.pod5` files in the specified GCS bucket and passes their paths to the basecalling task.
+
+??? task "`Dorado Basecalling`: Converts `POD5` files to 'SAM' files"
+    The basecalling task takes `POD5` files as input and converts them into 'SAM' format using the specified model. This step leverages GPU acceleration for efficient processing.
```

Comment:
Might be good to mention that barcode trimming is purposefully disabled in the basecalling step so that the demultiplexing step later in the wf can accurately demultiplex (and trim barcodes)